client/components/nux-notice/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import Gridicon from 'gridicons';
+import { translate as __ } from 'i18n-calypso';
+
+const NuxNotice = ( { children, onDismiss } ) => {
```

Comment:
We use this just once. Maybe it's not worth its own component? Probably you wanted to separate the dismissal logic? Can we use a Calypso component like `<Notice />` for that?